packages/validation-manager/src/TracerResultParser.ts

```diff
@@ -406,10 +413,16 @@ export function tracerResultParser (
   // return list of contract addresses by this UserOp. already known not to contain zero-sized addresses.
   const addresses = tracerResults.callsFromEntryPoint.flatMap(level => Object.keys(level.contractSize))
   const storageMap: StorageMap = {}
+  const transientStorageMap: StorageMap = {}
   tracerResults.callsFromEntryPoint.forEach(level => {
     Object.keys(level.access).forEach(addr => {
       storageMap[addr] = storageMap[addr] ?? level.access[addr].reads
     })
   })
-  return [addresses, storageMap]
+  tracerResults.callsFromEntryPoint.forEach(level => {
```

Comment:
the return value of addresses, storageMap is there to build conditionalRpc. We don't need transient storage for that.